snooty/diagnostics.py

```diff
@@ -331,3 +331,31 @@ def __init__(
     ) -> None:
         super().__init__(f"Could not locate toctree entry {entry}", start, end)
         self.entry = entry
+
+
+class MissingTab(Diagnostic):
+    severity = Diagnostic.Level.error
+
+    def __init__(
+        self,
+        tabs: Set[str],
+        start: Union[int, Tuple[int, int]],
+        end: Union[None, int, Tuple[int, int]] = None,
+    ) -> None:
+        super().__init__(
+            f"The following tab(s) did not appear in every tabset: {tabs}", start, end,
```

Comment:
`tabset` might be a bit jargon-y -- perhaps `one or more set of tabs on this page was missing the following required tabs:`